tutorials/vaultwarden-serverless.mdx

```diff
@@ -0,0 +1,126 @@
+     
+## Creating a serverless SQL Database
+
+Vaultwarden relies on a database to store all password data securely. The database stores encrypted passwords, associated metadata, and user data. Serverless SQL Databases adapt to your workloads automatically thanks to autoscaling, and can scale down to zero to reduce costs.
+
+1. Follow [this procedure](/serverless-sql-databases/how-to/create-a-database/) to create a Serverless SQL Database.
+
+2. On the **Overview** page of the database you created, click **Connect application**.
+
+3. To connect to the database you can either: 
+    - use an existing API secret key, by copying the **Connection string** and replacing the placeholder with your secret key.
+    - create a new API key, by clicking **Generate new secret key**, and by copying the newly created **Connection string**.
+
+    Your connection string should look like the following:
+    ```
+    postgresql://example-f3e2-4e5d-8e51-0fd3299d5d70:example-9ee1-43da-9231-0a351bf06139@example-8324-4d44-b03d-b6115096d2c7.pg.sdb.fr-par.scw.cloud:5432/serverless-sqldb-example-database?sslmode=require
+    ```
+    <Message type="note">
+    Store your connection string securely, as it will be needed in the next steps.
+    </Message>
+
+## Creating a Serverless Container
+
+1. [Create a Serverless Containers namespace](/serverless-containers/how-to/create-manage-delete-containers-namespace/#creating-a-serverless-containers-namespace), then click **Create namespace and add container** to directly open the Serverless Containers creation menu.
+
+2. Set the following parameters for the **image** of your container:
+    - Image from an external registry
+    - Image URL: `vaultwarden/server:latest`
+    - Port: `80`
+2. Set the following **resources**:
+    - 1000 mVCPU
+    - 2048 MB RAM
+3. Set the following **autoscaling** parameters:
+    - Minimum scale of `0` (set a value of 1 or more to avoid [cold starts](/serverless-containers/concepts/#cold-start))
+    - Maximum scale of `1` or more, according to your workload
+4. Set the following **environment variables**:
+    - `DATA_FOLDER`	= `/tmp`
+    - `ORG_ATTACHMENT_LIMIT`	= `0`
+    - `SIGNUPS_ALLOWED` = `true`
+    - `USER_ATTACHMENT_LIMIT`	= `0`
+5. Set the following ¨¨**secrets**:
+    - `DATABASE_URL` = the connection string retrieved  during the [previous step] (#creating-a-serverless-sql-database)
+    - `ADMIN_TOKEN` = the authentication token retrieved by following [This procedure](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page)
+6. Click **Deploy container**. The container **Overview** tab displays.
+
+## Finalizing the container configuration
+
+Once your container is deployed, you need to make a few adjustments before being able to fully use your Vaultwarden server.
+
+1. Access the **Settings** tab of your container.
+
+2. In the **Data** tab of the **Advanced options** section, add or update the following environment variables:
+    - `SIGNUPS_ALLOWED` =	`false`
```

Comment:
for singups you can mark is as optional, once the customer created his account, if he does not want unwanted singup he an define the variable